packages/Metric/src/Score.tsx

```diff
@@ -64,7 +67,9 @@ const Score: React.FunctionComponent<ScoreProps> = ({
             })}
         />
     );
-    const postFix = absoluteValue === 1 ? " pt" : " pts";
+    const postFix = absoluteValue === 1 ? 
```

Comment:
really really not important, but i'm like 90% sure that in english, 0 or multiple is plural, and 1 is singular. 
in french, it's 0 or 1 singular, multiple is plural.

Reply:
ugh that's unfortunate.. :( 
if the messages were working properly, we are able to put plural rules inside the intl file :(